src/daemon/absorber_test.cc

```diff
@@ -8,15 +8,24 @@ namespace daemon {
 
 namespace {
 
+static const ui32 kEmptyEmitterVersion = 0u;
+static const ui32 kBadEmitterVersion = 1u;
```

Comment:
This constant should be replaced with something like `proto::Remote::default().version() - 1`